src/utils/format_graphite/format_graphite.c

```diff
@@ -35,15 +35,23 @@
 /* Utils functions to format data sets in graphite format.
  * Largely taken from write_graphite.c as it remains the same formatting */
 
-static int gr_format_values(strbuf_t *buf, metric_t const *m, gauge_t rate) {
-  if (m->family->type == DS_TYPE_GAUGE)
-    return strbuf_printf(buf, GAUGE_FORMAT, m->value.gauge);
-  else if (rate != -1)
-    return strbuf_printf(buf, "%f", rate);
```

Comment:
This case seems to be missing in the new code, is this intended?

Reply:
Yes. This code previously (erroneously) used `DS_TYPE_…` instead of the values defined by the `metric_type_t` enum, e.g. `METRIC_TYPE_GAUGE`. This has been fixed in the process.